tools/clang/lib/SPIRV/DeclResultIdMapper.cpp

```diff
@@ -100,15 +92,15 @@ uint32_t DeclResultIdMapper::getDeclResultId(const NamedDecl *decl) {
 uint32_t DeclResultIdMapper::createFnParam(uint32_t paramType,
                                            const ParmVarDecl *param) {
   const uint32_t id = theBuilder.addFnParam(paramType, param->getName());
-  astDecls[param] = {id, spv::StorageClass::Function};
+  astDecls[param] = {id, spv::StorageClass::Function, -1};
```

Comment:
It will soon be killed in the [next next commit](https://github.com/antiagainst/DirectXShaderCompiler/commit/76659e48652f945b1a743ad1d9b3a83b559f4cbe). So please wait for a short while. :)